src/test/java/org/jenkinsci/plugins/gitclient/verifier/AcceptFirstConnectionVerifierTest.java

```diff
@@ -130,7 +130,7 @@ public void testVerifyServerHostKeyWhenHostnameWithoutPort() throws Exception {
             return; // Test fails with connection timeout on ci.jenkins.io kubernetes agents
         }
         String hostKeyEntry =
-                "github.com ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBEmKSENjQEezOmxkZMy7opKgwFB9nkt5YRrYMjNuG5N87uRgg6CLrbo5wAdT/y6v0mKV0U2w0WZ2YB/++Tpockg=]";
+                "github.com ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBEmKSENjQEezOmxkZMy7opKgwFB9nkt5YRrYMjNuG5N87uRgg6CLrbo5wAdT/y6v0mKV0U2w0WZ2YB/++Tpockg=";
```

Comment:
@MarkEWaite  LOL I just  wasted few hours only because of a ridiculous typo but couldn't see it :rofl:

Reply:
@MarkEWaite haha no worries. I'm just complaining about myself not seeing this quicker. 
two pairs of eyes are not enough